### DISCUSSION TITLE
Support ottai

### DISCUSSION THREAD

**Discussion by djscith** [NightscoutFoundation/xDrip#3392] [General]
when is support for ottai sensors implemented?

**Comment by Navid200** [comment]
Please provide a link.

**Reply by WarmBooter** [reply]
https://geoip.synapsor.com/product

**Reply by addrandomuser** [reply]
https://ottai.com/en/product

**Comment by SimonXuku** [comment]
This sensor only sells for $18-$20,it's cheap and accurate.
A little slow than G6.

**Comment by Kamusoo** [comment]
I'm wearing this sensor right now, maybe I can help integrate it into xdrip

**Reply by Navid200** [reply]
Do you know if the manufacturer is open to discussing it with us and supporting the integration?

**Reply by Kamusoo** [reply]
As far as I know, integration with AndroidAPS has already appeared in the Chinese version of the application, so I think the manufacturer is ready to cooperate

**Reply by Navid200** [reply]
How can I establish communication with the manufacturer?

**Reply by Kamusoo** [reply]
on their website there is an email service@ottai.com
I communicated with them about a normal translation into my native language

**Reply by Navid200** [reply]
I will ask them, thanks

**Reply by Kamusoo** [reply]
Thank you too, and could you keep us updated?

**Reply by Kamusoo** [reply]
I also have almost direct contact with the developer, if you want to know something, ask, I will try to tell you

**Reply by Kamusoo** [reply]
I was also able to ask for the email of the Ottai developers, I think this will make it a little easier to contact them and communicate 
ottai-dev@ottai.com

**Reply by Kamusoo** [reply]
Hello, how are the negotiations with the developer?

**Comment by darkmic** [comment]
Is support for this sensor planned and what might influence the decision?

**Comment by qq254344379** [comment]
You can try to use their already configured AAPS source, which is whether this can be rewritten to XD to facilitate everyone to use the AAPS of the main line

**Comment by CrYaZer** [comment]
Hello, is support for the ottai sensor planned? Thank you for supporting the Ottai companion app. Still, I wonder if there will be support for the sensor itself?
